zappa/utilities.py

```diff
@@ -592,3 +592,11 @@ def merge_headers(event):
     for h in multi_headers.keys():
         multi_headers[h] = ", ".join(multi_headers[h])
     return multi_headers
+
+
+def validate_json_seralizable(function_args: dict) -> None:
+    try:
+        json.dumps(function_args)
+    except Exception:
+        LOG.error("Arguments expected to be JSON seralizable!")
+        raise  # re-raise initial error
```

Comment:
Why not create a custom exception class here?
- easier to understand for humans
- could spare logging the error because the string `arguments expected...` can be use in the exception 

basic example here: https://github.com/zappa/Zappa/blob/fff5ed8ad2ee071896a94133909adf220a8e47d9/zappa/asynchronous.py#L127

`class UnserializableJsonError(Error)` or similar

Reply:
Added `zappa.utilities.UnserializableJsonError`

Reply:
@hellno updated.